### ISSUE TITLE
Create a new template for Compose

### ISSUE THREAD

**Issue by Tuubz** [nimblehq/android-templates#333]
## Why

The template currently consists of both `XML` & `Compose`.

For the sake of simplicity, we will have a separate template for XML and another one for Compose. 

## Acceptance Criteria

- Create new folder `template-compose` copied from `template`
- Change package name to `co.nimblehq.template.compose`
- Update app name to `Template Compose`
- Update the workflows for `template-compose`

**Note**: Removing the code of XML will be done in a different issue (#335)

**Comment by luongvo** [comment]
@Tuubz @chornerman I think we should do #334 in this task also. Clone the `template` into the new `template-compose` and remove unnecessary things from the XML method in just 1 PR. I don't think we should introduce new file added and then remove them in a separate PR.

**Comment by Tuubz** [comment]
@chornerman @luongvo

 Let's not do #333 & #334 together in one pull request, otherwise, it will be difficult to see what changes have been made 🙏 
 
IMO, if we do it in two pull requests it will be clear to see what files of XML are being removed from `template-compose`.

What do you think? 🙏

**Comment by luongvo** [comment]
@Tuubz lgtm 👍

**Comment by Wadeewee** [comment]
@Tuubz, I think you need to update the `AC` a bit of the first bullet point

> - Create new folder template-compose copied from template

**Comment by Tuubz** [comment]
@Wadeewee Good morning 👋 

What update do you suggest I should make? 🙏

**Comment by Wadeewee** [comment]
> What update do you suggest I should make? 🙏

@Tuubz To work on this, Should we branch out from `chore/332-rename-the-current-template-to-xml` because we don't have `template` once we implemented [#332](https://github.com/nimblehq/android-templates/issues/332) 🤔

**Comment by Wadeewee** [comment]
Oh, wait, question @Tuubz Why don't we rename `sample-compose` instead of creating a new one for `template-compose` 🤔

**Comment by Tuubz** [comment]
@Wadeewee @chornerman  The idea originally was to do #333 first, but I'm fine to do it after #332 too 👍 

> Why don't we rename sample-compose instead of creating a new one for template-compose 🤔

Hmmm, `sample-compose` is for sample purposes, `template-compose` is used to generate a new project.

Please let me know if I misunderstood you question 🙇
